content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -190,3 +151,49 @@ To ignore multiple codes for a resource, separate each code with a comma:
 {{< text bash >}}
 $ kubectl annotate deployment my-deployment galley.istio.io/analyze-suppress=IST0107,IST0002
 {{< /text >}}
+
+## Helping us improve this tool
+
+We're continuing to add more analysis capability and we'd love your help in identifying more use cases.
+If you've discovered some Istio configuration "gotcha", some tricky situation that caused you some
+problems, open an issue and let us know. We might be able to automatically flag this problem so that
+others can discover and avoid the problem in the first place.
+
+To do this, [open an issue](https://github.com/istio/istio/issues) describing your scenario. For example:
+
+- Look at all the virtual services
+- For each, look at their list of gateways
+- If some of the gateways don’t exist, produce an error
+
+We already have an analyzer for this specific scenario, so this is just an example to illustrate what
+the kind of information you should provide.
+
+## Q&A
+
+- **What Istio release does this tool target?**
+
+      Like other `istioctl` tools, we generally recommend using a downloaded version that matches the version deployed in your cluster.
+
+      For the time being, analysis is generally backwards compatible, so that you can e.g. run the 1.4 version of `istioctl analyze` against a cluster running Istio 1.1 and expect to get useful feedback. Analysis rules that are not meaningful with an older Istio release will be skipped.
```

Comment:
```suggestion
      For the time being, analysis is generally backwards compatible, so that you can, for example, run the 1.4 version of `istioctl analyze` against a cluster running Istio 1.1 and expect to get useful feedback. Analysis rules that are not meaningful with an older Istio release will be skipped.
```